app/data/tests/test_views.py

```diff
@@ -156,6 +166,19 @@ def test_arbitrary_filters(self):
         response_data2 = json.loads(self.admin_client.get(url2).content)
         self.assertEqual(len(response_data2), 2)
 
+    def test_created_by(self):
+        url = '/api/records/?details_only=True'
+
+        response_data = json.loads(self.admin_client.get(url).content)
+        record3_result = [
+            r for r in response_data['results']
+            if r['uuid'] == str(self.record3.uuid)
+        ][0]
```

Comment:
Performance-wise for this kind of use it's probably basically equivalent and I'm not sure if it's necessarily clearer or not, but similar to `all()` you can use `next()` with a generator expression to do tasks like this instead of a list comprehension that we take the first element from. It's pretty minor, but in broader use has the advantage that it stops on the first element it finds, instead of parsing the whole list and then discarding any extra results.